hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -629,6 +638,7 @@ public OzoneOutputStream createKey(
         .setType(HddsProtos.ReplicationType.valueOf(type.toString()))
         .setFactor(HddsProtos.ReplicationFactor.valueOf(factor.getValue()))
         .addAllMetadata(metadata)
+        .setAcls(getAclList())
```

Comment:
Should we support createKey with acls as parameter instead of always set default and modify with additional xxxAcl RPC calls?

Reply:
I think default acls more sensible as this will add current user and its group. This makes behavior for default ACLs standard. While parametrized different users may set it with different approaches.

Reply:
Had offline discussion with @xiaoyuyao , we will overload this api in separate jira.